vtex-apps/product-list
# CHK-261: Fix price not shown as free label when it is zero

#### What problem is this solving?

Updates the condition to show the item `sellingPrice` to compare it with `null` instead of checking only the truthiness of the value.

#### How to test it?

[Workspace](https://lucas--checkoutio.myvtex.com/checkout/cart/add/?sku=282&qty=1&seller=1&sc=1&utm_source=itemgratis).

The item should appear with the "free" label, instead of showing a zero in the place of the price.

#### Screenshots or example usage:

![image](https://user-images.githubusercontent.com/10223856/91210283-59655800-e6e3-11ea-9158-a6b62d7a8c3e.png)

#### Describe alternatives you've considered, if any.

N/A

#### Related to / Depends on

N/A
